proto/v1/bookmark_service.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package bytebase.v1;
+
+import "google/api/annotations.proto";
+import "google/api/client.proto";
+import "google/api/field_behavior.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/wrappers.proto";
+
+import "v1/common.proto";
+
+option go_package = "generated-go/v1";
+
+service BookmarkService {
+  // CreateBookmark creates a new bookmark.
+  rpc CreateBookmark(CreateBookmarkRequest) returns (Bookmark) {
+    option (google.api.http) = {
+      post: "/v1/bookmark"
+      body: "*"
+    };
+  }
+  
+  // DeleteBookmark deletes a bookmark.
+  rpc DeleteBookmark(DeleteBookmarkRequest) returns (google.protobuf.Empty) {
+    option (google.api.http) = {
+      delete: "/v1/bookmark/{id=*}"
+    };
+  }
+
+  // ListBookmark lists bookmarks.
+  rpc ListBookmark(ListBookmarkRequest) returns (BookmarkList) {
+    option (google.api.http) = {
+      get: "/v1/bookmark/{user_name=users/*}"
```

Comment:
May be put the `user_name` in the query?

Reply:
How about `get: "/v1/{user_name=users/*}/bookmark"`? @ecmadao @d-bytebase

Reply:
SGTM